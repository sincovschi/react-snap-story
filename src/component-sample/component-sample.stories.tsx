import React from 'react'
import { ComponentSample } from './component-sample'

export default {
  title: 'Sample component'
}

export function ComponentSampleStory () {
  return <ComponentSample />
}
