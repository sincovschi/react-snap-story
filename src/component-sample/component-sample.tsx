import React from 'react'

// import "./component-sample.module.scss";

export interface ComponentSampleProps {}

export function ComponentSample (props: ComponentSampleProps) {
  return (
    <div>
      <h1>Welcome to common-components!</h1>
    </div>
  )
}
