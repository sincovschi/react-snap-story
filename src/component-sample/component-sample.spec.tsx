import React from 'react'
import { render } from '@testing-library/react'
import { ComponentSampleStory } from './component-sample.stories'

describe('ComponentSample', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<ComponentSampleStory />)
    expect(baseElement).toMatchSnapshot()
  })
})
