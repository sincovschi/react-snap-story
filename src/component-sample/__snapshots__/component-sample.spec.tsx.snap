// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentSample should render successfully 1`] = `
<body>
  <div>
    <div>
      <h1>
        Welcome to common-components!
      </h1>
    </div>
  </div>
</body>
`;
